frontend/src/modules/scribble/view/Scribble.vue

```diff
@@ -21,6 +21,19 @@
                 </b-row>
                 <Modal />
             </b-container>
+            <b-modal
```

Comment:
What if we moved this into the modal container and the logic to make it show up too?

Reply:
So this was a conscious decision not to do so, because I was thinking the freedom to show how the results would be displayed should be up to the caller, i.e. it might not be a modal.